UIKitTask/Sources/Scenes/PostsScene/PostsViewController.swift

```diff
@@ -0,0 +1,184 @@
+// PostsViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Стартовый экран, список постов , сторис, рекомендаций
+final class PostsViewController: UIViewController {
+    // MARK: Types
+
+    private enum TableSections {
+        case firstPost
+        case stories
+        case posts
+        case recommendations
+    }
+
+    // MARK: Visual Components
+
+    private let appLogoBarImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.contentMode = .scaleAspectFill
+        imageView.image = .appLogo
+        return imageView
+    }()
+
+    private let messagesBarButton: UIButton = {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.contentMode = .scaleAspectFill
+        button.setImage(.messagesButtonBar, for: .normal)
+        return button
+    }()
+
+    private lazy var mainTableView: UITableView = {
+        let tableView = UITableView()
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.backgroundColor = .clear
+        tableView.register(StoriesCell.self, forCellReuseIdentifier: StoriesCell.identifier)
+        tableView.register(RecommendationCell.self, forCellReuseIdentifier: RecommendationCell.identifier)
+        tableView.register(PostCell.self, forCellReuseIdentifier: PostCell.identifier)
+        tableView.dataSource = self
+        tableView.separatorStyle = .none
+        return tableView
+    }()
+
+    private lazy var refreshTableControl: UIRefreshControl = {
+        let refreshControl = UIRefreshControl()
+        refreshControl.translatesAutoresizingMaskIntoConstraints = false
+        refreshControl.addTarget(self, action: #selector(refreshTable), for: .valueChanged)
+        return refreshControl
+    }()
+
+    // MARK: Private Properties
+
+    private let dataBase = DataStorageService.shared
```

Comment:
```suggestion
    private let dataBase = DataStorageService.shared
```
dataBase это модель а не база данных